editor.planx.uk/src/ui/shared/DataTable/utils.tsx

```diff
@@ -111,3 +112,8 @@ export const getColumnFilterType = (columnType?: ColumnRenderType) => {
       return undefined;
   }
 };
+
+/**
+ * Format date times to a standard format, e.g 31/12/25 14:40:45
+ */
+export const dateFormatter = (value: string) => format(new Date(value), "dd/MM/yy HH:mm:ss");
```

Comment:
There is GDS guidance on date and time formatting, but I'd consider these rules we should apply in applicant-facing situations, not internally.

Elsewhere in PlanX I think we just display time as "13 hours ago" etc - if there are other instances we need to standardise please point them out.

https://www.gov.uk/guidance/style-guide/a-to-z#times

https://www.gov.uk/guidance/style-guide/a-to-z#dates